columnflow/plotting/plot_util.py

```diff
@@ -396,22 +370,49 @@ def apply_density_to_hists(hists: dict, density: bool | None = False) -> dict:
     return hists
 
 
-def remove_residual_axis(hists: dict, ax_name: str, max_bins: int = 1) -> dict:
+def _remove_residual_axis(
```

Comment:
Maybe just a bit more verbosity to understand the differences between the two functions, e.g.
```suggestion
def remove_residual_axis_single_hist(
```